frontend/src/components/housing/submission/SubmissionListNavigator.vue

```diff
@@ -158,7 +158,7 @@ function updateQueryParams() {
     :sort-order="pagination.order"
     paginator-template="RowsPerPageDropdown CurrentPageReport PrevPageLink NextPageLink "
     current-page-report-template="{first}-{last} of {totalRecords}"
-    :rows-per-page-options="[10, 20, 50]"
+    :rows-per-page-options="[10, 20, 50, submissions?.length as number]"
```

Comment:
Is this meant to behave as a "show all rows" option? The value on this looks like it can change, which could be jarring, or slightly strange if there happen to be exactly 10, 20 or 50 records present.

Reply:
Yes. Request from the business to have an "all" option. Made this dynamic and only appear when needed.